docker/Vagrantfile

```diff
@@ -0,0 +1,102 @@
+#========================================================================
+# Author: Edoardo Pasca, Ben Thomas
+# Copyright 2018 University College London
+# Copyright 2018-2019 Science Technology Facilities Council
+#
+# This file is part of the CCP PETMR Synergistic Image Reconstruction Framework (SIRF) virtual machine.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#=========================================================================
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "bento/ubuntu-18.04"
+  config.vm.synced_folder '.', '/vagrant', disabled: true
+  config.vm.define "docker"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  # Forward port for jupyter notebooks
+  config.vm.network "forwarded_port", guest: 8888, host: 8889
```

Comment:
`guest` should be 9999 as jupyter on the docker instance responds to port 8888, which is forwarded to 9999.

Host: 8889 -> VirtualBox 9999 -> docker 8888